docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -0,0 +1,338 @@
+        ```
+    1. It created a Knative Eventing Trigger to route certain events to the helloworld-python application. Make sure that Ready=true
+        ```shell
+        kubectl --namespace knative-samples get trigger helloworld-python
+        ```
+## Send and verify CloudEvents
+Once you have deployed the application and verified that the namespace, sample application and trigger are ready, let's send a CloudEvent.
+
+### Send CloudEvent to the Broker
+We can send an http request directly to the [Broker](../../../broker-trigger.md) with correct CloudEvent headers set.
+
+   1. Deploy a curl pod and SSH into it
+      ```shell
+      kubectl --namespace knative-samples run curl --image=radial/busyboxplus:curl -it
+      ```
+  1. Run the following in the SSH terminal
+      ```shell
+      curl -v "default-broker.knative-samples.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: 536808d3-88be-4077-9d7a-a3f162705f79" \
+      -H "Ce-specversion: 0.3" \
+      -H "Ce-Type: dev.knative.samples.helloworld" \
+      -H "Ce-Source: dev.knative.samples/helloworldsource" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello World from the curl pod."}'
+
+      exit
+      ```
+### Verify that event is received by helloworld-python app
+Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
+  1. Display helloworld-python app logs
+      ```shell
+      kubectl --namespace knative-samples logs -l app=helloworld-python --tail=50
+      ```
+      You should see something similar to:
+      ```shell
+      Event received. Context: Context Attributes,
+        specversion: 0.3
+        type: dev.knative.samples.helloworld
+        source: dev.knative.samples/helloworldsource
+        id: 536808d3-88be-4077-9d7a-a3f162705f79
+        time: 2019-10-04T22:35:26.05871736Z
+        datacontenttype: application/json
+      Extensions,
+        knativearrivaltime: 2019-10-04T22:35:26Z
+        knativehistory: default-kn2-trigger-kn-channel.knative-samples.svc.cluster.local
+        traceparent: 00-971d4644229653483d38c46e92a959c7-92c66312e4bb39be-00
+
+      Hello World Message "Hello World from the curl pod."
+      Responded with event Validation: valid
+      Context Attributes,
+        specversion: 0.2
+        type: dev.knative.samples.hifromknative
+        source: knative/eventing/samples/hello-world
+        id: 37458d77-01f5-411e-a243-a459bbf79682
+      Data,
+        {"msg":"Hi from Knative!"}
+
+      ```
+  Play around with the CloudEvent attributes in the curl command and the trigger specification to understand how [Triggers work](../../../broker-trigger.md#trigger).
+
+## Verify reply from helloworld-python app
+`helloworld-python` app replies back with an event of `type= dev.knative.samples.hifromknative`, and `source=knative/eventing/samples/hello-world`. This event enters the eventing mesh via the Broker and can be delivered to other services using a Trigger
```

Comment:
Again, "The...app"
Also, be consistent when using backticks, formatting, capitalization for all naming. The app is referred to in different ways throughout the doc.

Reply:
Use "through", not "via"